### DISCUSSION TITLE
Sanity Projection

### DISCUSSION THREAD

**Discussion by ErdanThren** [deltagreen-foundryvtt/delta-green-foundry-vtt-system#114] [Ideas]
Would it be possible to have it so that when a sanity check is rolled it asks if the player wants to project some of that sanity damage, lowering bonds, WP etc?

**Comment by TheLastScrub** [comment]
It is possible but I'm not sure you would want it on every SAN roll, since many are not necessarily for SAN loss (especially for something like Impossible Landscapes). It might be something that could be added to a chat card. For instance the Handler could roll the damage, and we could add a button under the part of the card where the results currently are that triggers a SAN roll:

![image](https://github.com/TheLastScrub/delta-green-foundry-vtt-system-unofficial/assets/67403321/0e0d942e-d3c2-4f9b-9522-f99040a60818)

This works well for something like Pathfinder 2e, but possibly less well for Delta Green due to not having tokens directly on the map much. I'd think this would be the cleanest way to do it, but I haven't had a chance to play around with it much recently.
